exercises/binary_converter.cpp

```diff
@@ -5,3 +5,36 @@
   Insert first number: 8
   The binary number is: 1000
 */
+
+#define MAX_BITS 32
+#include <stdio.h>
+#include <limits.h>
+
+bool bits[MAX_BITS];
+
+int main()
+{
+  int N;
+  printf("Please enter a positive number below: ");
+  scanf("%d", &N);
+
+  if (N <= 0)
+  {
+    printf("Please enter a positive number below %d\n", INT_MAX);
+    return 1;
+  }
+
+  int j = MAX_BITS - 1;
+
+  while (N > 0)
+  {
+    bits[j--] = N % 2;
+    N >>= 1;
+  }
+
+  for (int i = 0; i < MAX_BITS; i++)
+    printf("%d", bits[i]);
+  printf("\n");
+
+  return 0;
```

Comment:
Potresti rendere più esplicito il valore di ritorno del **main** aggiungendo in cima al codice, subito dopo gli "include"

```c++
#define EXIT_SUCCESS 0
#define EXIT_FAILURE 1
```

e al posto di "return 0;" o "return 1;"

```c++
// return 1;
return EXIT_FAILURE;
// return 0;
return EXIT_SUCCESS;
```